gateway/gateway/urls.py

```diff
@@ -0,0 +1,35 @@
+"""gateway URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/4.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+from rest_framework import routers
+
+from api.views import ProgramViewSet, KeycloakLogin, JobViewSet, KeycloakUsersView
+
+router = routers.DefaultRouter()
+router.register(r"programs", ProgramViewSet)
+router.register(r"jobs", JobViewSet)
+
+
+urlpatterns = [
+    path("", include(router.urls)),
+    path("dj-rest-auth/", include("dj_rest_auth.urls")),
+    path("dj-rest-auth/keycloak/", KeycloakLogin.as_view(), name="keycloak_login"),
+    path("dj-rest-auth/keycloak/login/", KeycloakUsersView.as_view()),
+    path("accounts/", include("allauth.urls")),
+    path("api-auth/", include("rest_framework.urls")),
```

Comment:
Similar to a previous comment. Default end-points from django can be removed but I will analyze it.